MIGRATION.md

```diff
@@ -1,5 +1,43 @@
 # Migration guide
 
+## Migrating to JSON Forms 3.2
+
+### Material Renderers using Outlined Inputs
+
+JSON Forms has been updated to use the `outlined` input variant as the default, aligning more closely with the default style of Material UI.
+To use the `standard` input variant, as in previous versions of JSON Forms, it is possible to provide JSON Forms with a theme provider:
```

Comment:
```suggestion
If you would like to use the `standard` input variant, as was default in previous versions of JSON Forms, then this can be accomplished using the Material UI `ThemeProvider`:
```